### PR TITLE
Add Jupyterhub to osc playground.

### PR DESCRIPTION
**Pull request by HumairAK**
Add Jupyterhub for OS-C playground. A separate environment is needed to allow for experimentation with JH itself if needed, and some flexibility.

### CODE REVIEW

File: kfdefs/overlays/moc/zero/osc-playground/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+patchesJson6902:
+  - patch: |
+      - op: add
+        path: /spec/applications/2/kustomizeConfig/overlays/-
+        value: storage-class
+      - op: add
+        path: /spec/applications/2/kustomizeConfig/parameters/-
+        value:
+          name: storage_class
+          value: ocs-storagecluster-cephfs
```
Comment (tumido) [inline kfdefs/overlays/moc/zero/osc-playground/kustomization.yaml:18]: Just noticed.. maybe you wanna use NFS?

Comment (HumairAK) [reply kfdefs/overlays/moc/zero/osc-playground/kustomization.yaml:18]: yes I do good catch

Comment (HumairAK) [reply kfdefs/overlays/moc/zero/osc-playground/kustomization.yaml:18]: I copied it from stage...which should also probably be nfs

Comment (HumairAK) [reply kfdefs/overlays/moc/zero/osc-playground/kustomization.yaml]: updated

Comment (4n4nd) [reply kfdefs/overlays/moc/zero/osc-playground/kustomization.yaml]: we can just remove it and let it use the default storageclass (which currently is the nfs)

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**